src/extensions/surveys.ts

```diff
@@ -457,6 +460,9 @@ export const addCancelListeners = (
                 $survey_name: surveyEventName,
                 $survey_id: surveyId,
                 sessionRecordingUrl: posthog.get_session_replay_url?.(),
+                $set: {
+                    [`$survey_dismiss/${surveyId}`]: true,
```

Comment:
I think this was still in draft mode when I looked at it yesterday :(  but this is probably better as `survey_dismissed`

Reply:
Like one can probably gather what it's meant to do even with just "survey dismiss" but we do `feature_flag_called` and it makes a lot more sense than if we did `feature_flag_call`